builder/evaluator.go

```diff
@@ -74,7 +74,7 @@ func init() {
 	}
 }
 
-// internal struct, used to maintain configuration of the Dockerfile's
+// Builder is an internal struct, used to maintain configuration of the Dockerfile's
```

Comment:
Yeah, I feel confused too, maybe we can have a deep look later, and focus on golint problems in this PR, right?